Cloud Bigtable

adithya1811 opened the pull request:
1 policy with 100 aruguments

adithya1811 committed:
Cloud Bigtable

1 policy with 100 aruguments

jashan-lefty reviewed (changes requested):
Hi mate, Hope you are doing well!

I had a look at the PR, I think you kinda misunderstood the policy writing process. No stress, you have time to resubmit the PR. I want you to have a look at demo videos and create clean working policies. Probably look at the main branch and see how policies are written. If you still need some assisstance please reach out to the fellow team members, leaders or Yotam for clarification.

adithya1811 committed:
Update Policy.rego

adithya1811 commented:
Hi Mate,
I have seen the videos and created a policy based on my understanding. Requesting you to review the same.

adithya1811 committed:
Update vars.rego

adithya1811 committed:
Update vars.rego

jashan-lefty commented:
Hi Adithya, I am not sure if you're getting the method of writing policies. Kindly reach out to one of the seniors for help. You are expected to submit working policies in order to pass.

adithya1811 committed:
Revised Version

Hi I have revised as per my understanding. Trying to connect with a senior for better understanding

adithya1811 committed:
Feedback addressed policies

jashan-lefty commented:
Hi mate, you have conflicts for all the files. Could you restore the previous version of the branch to solve the errors?

adithya1811 committed:
Revision Done

Hello I have done the revision mentioned

adithya1811 commented:
Hi Mate,
I have done the changes mentioned. beyondcorp still shows some conflicts. I haven't done any changes in beyondcorp's folder. I have only worked on Bigtable's folder

adithya1811 committed:
Update policy.rego
